pyproject.toml

```diff
@@ -52,6 +52,8 @@ dbt-redshift = {version="^1.4.0", optional=true}
 dbt-postgres = {version="^1.4.0", optional=true}
 dbt-bigquery = {version="^1.4.0", optional=true}
 dbt-metadata-client = {version="^0.1.0", optional=true}
+cryptography = "^39.0.1"
+pyopenssl = "^23.0.0"
```

Comment:
OK, this buys us time to update the things that depend on these things, so let's do it. Could you add a comment to indicate that we don't need these, they're just included to ensure we stay ahead of vulnerabilities in transitive package installations?

I think the reason this works is because the cryptography version we were installing was pinned to an optional dependency of some package.